src/lightbox/tp-lightbox-trigger.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Internal dependencies.
+ */
+import { TPLightboxElement } from './tp-lightbox';
+
+/**
+ * TP Lightbox Trigger.
+ */
+export class TPLightboxTriggerElement extends HTMLElement {
+	/**
+	 * Constructor.
+	 */
+	constructor() {
+		super();
+
+		// Events.
+		this.querySelector( 'button' )?.addEventListener( 'click', this.trigger.bind( this ) );
+	}
+
+	/**
+	 * Trigger the lightbox.
+	 */
+	trigger(): void {
+		// Get lightbox ID and template.
+		const lightboxId: string | null = this.getAttribute( 'lightbox' );
+		const template: HTMLTemplateElement | null = this.querySelector( 'template' );
+
+		// We can't proceed without them.
+		if ( ! lightboxId || ! template ) {
+			return;
+		}
+
+		// Get the lightbox.
+		const lightbox: TPLightboxElement | null = document.querySelector( `#${ lightboxId.toString() }` );
+		if ( ! lightbox ) {
+			return;
+		}
+
+		// Check to see if we have a group.
+		const group: string = this.getAttribute( 'group' ) ?? '';
```

Comment:
@junaidbhura Let's change this to 
`const group: string = this.getAttribute( 'group' ) || '';`